actions/cursor-deploy/README.md

```diff
@@ -72,50 +83,46 @@ on:
                 required: false
 jobs:
     rollback:
-        runs-on: ubuntu-20.04
+        runs-on: ubuntu-22.04
         steps:
-            - uses: actions/checkout@v2
-              with:
-                  fetch-depth: 10
             - uses: aws-actions/configure-aws-credentials@v1
               with:
                   aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
                   aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
             - name: Update Cursor File
-              uses: 'pleo-oss/pleo-spa-cicd/actions/cursor-deploy@v1'
+              uses: pleo-oss/pleo-spa-cicd/actions/cursor-deploy@v5
               with:
-                  bucket_name: my-origin-bucket
-                  rollback_commit_hash: ${{ github.event.inputs.sha }}
-                  deploy_mode: rollback
+                  bucket-name: my-origin-bucket
+                  deploy-mode: rollback
+                  rollback-jump: ${{ github.event.inputs.jump }}
```

Comment:
Do we suggest that this is the preferred workflow for rolling back? I imagine that it can be hard to know which version to jump back to.